docs/deploying_server.md

```diff
@@ -71,13 +71,42 @@ zebra
 ```
 If everything is set up correctly, you will see 'zebra' prediction in the output.
 
-## Deploying Model Server on Baremetal
+## Deploying Model Server on Baremetal (without docker)
+It is possible to deploy Model Server outside of docker container.
+To deploy Model Server on baremetal, use pre-compiled binaries for your target OS from latest release.
+Alternatively it is possible to build package from scratch:
 
-There are two ways to start the server:
+```bash
+git clone https://github.com/openvinotoolkit/model_server
+cd model_server
+```
+
+The `ovms.tar.gz` package will appear in `dist/ubuntu` or `dist/redhat` directory.  
+Unpack the package:
+
+```bash
+tar -xzvf dist/ubuntu/ovms.tar.gz
+```
+
+Install required libraries depending on the OS.  
+For Ubuntu 20.04:
+```bash
+apt update -y && apt install -y libpugixml1v5 libtbb2
```

Comment:
Right now I think that tbb is not required due to OV not containing HDDL plugin in 2022.3 release. I don't like that we have to copy the required libraries here from the OV.

We could mention here that the user is required to follow requirements coming from appropriate plugin devices (used by they) and then install OV. Otherwise this instruction will require changes from us for each release. None of the packages here are "pure" OVMS requirements and are just transitive requirements.

For now it is ok.

Reply:
As agreed during sync, leaving as is for this release.